src/api/graphapi.h

```diff
@@ -0,0 +1,417 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Apache License, Version 2.0,
+* modified with the Commons Clause restriction.
+*/
+
+#ifndef REDISGRAPH_API_H
+#define REDISGRAPH_API_H
+
+// #ifdef DREDIS_MODULE_TARGET
+
+// RG_VERSION: a single integer for comparing spec and version levels
+#define RG_VERSION(major, minor, sub) \
+    (((major)*1000ULL + (minor))*1000ULL + (sub))
```

Comment:
Why not use the same versioning logic as we have elsewhere?
`major * 10000 + minor * 100 + patch`